tests/integration/cartography/intel/aws/test_route53.py

```diff
@@ -1,5 +1,6 @@
 import cartography.intel.aws.route53
```

Comment:
This is the existing integ test file. Most tests still work without needing to change interface.